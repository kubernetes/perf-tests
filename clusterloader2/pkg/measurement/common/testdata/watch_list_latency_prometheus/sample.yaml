interval: 1m
input_series:
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="1"}
    values: 0 0 0 0 0 0 1 991 991 991 991
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="5"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="+Inf"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001

  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="1"}
    values: 0 0 0 0 0 0 1 991 991 991 991
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="5"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="+Inf"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001

  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="1"}
    values: 0 0 0 0 0 0 1 991 991 991 991
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="5"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="+Inf"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001

  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="1"}
    values: 0 0 0 0 0 0 1 701 701 701 701 701
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="1.5"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="+Inf"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001

  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="5"}
    values: 0 0 0 0 0 0 1 901 901 901 901 901
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="10"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001
  - series: apiserver_watch_list_duration_seconds{component="apiserver",group="storage.k8s.io",resource="pod",scope="namespace",version="v1",le="+Inf"}
    values: 0 0 0 0 0 0 1 1001 1001 1001 1001