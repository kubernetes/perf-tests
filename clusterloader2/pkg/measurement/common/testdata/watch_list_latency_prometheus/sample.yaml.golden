{
  "version": "v1",
  "dataItems": [
    {
      "data": {
        "Perc50": 713.980028,
        "Perc90": 1333.166666,
        "Perc99": 1483.316666
      },
      "unit": "ms",
      "labels": {
        "Group": "storage.k8s.io",
        "Resource": "pod",
        "Scope": "namespace",
        "Version": "v1"
      }
    }
  ]
}
